types/sinon/index.d.ts

```diff
@@ -309,7 +309,7 @@ declare namespace Sinon {
         /**
          * Returns the nth call.
          * Accessing individual calls helps with more detailed behavior verification when the spy is called more than once.
-         * @param n
+         * @param n Index of the spy call. This is zero based, so use three if you want the fourth call.
```

Comment:
can we _slightly_ change the wording to something like `Zero-based index of the spy call` i think the rest can easily be inferred

Reply:
Yes, of course. Should be sufficient.

Reply:
Can you do it? I have no access to my change at the moment.

Reply:
unfortunately don't have the permission to, but no rush. will approve it soon as you get chance to update

Reply:
Ok, thanks. I will do this in about 12 hours.

Reply:
Pushed my change.